frb_codegen/src/generator/dart/ty_delegate.rs

```diff
@@ -81,6 +99,18 @@ impl TypeDartGeneratorTrait for TypeDelegateGenerator<'_> {
                 }
                 IrTypeTime::Duration => "return wire2apiDuration(_wire2api_i64(raw));".to_owned(),
             },
+            #[cfg(feature = "uuid")]
+            IrTypeDelegate::Uuid => "return UuidValue.fromByteList(raw);".to_owned(),
+            #[cfg(feature = "uuid")]
+            IrTypeDelegate::Uuids => "
+            final count = raw.lengthInBytes / 16;
+            var buffer = List<UuidValue>.empty(growable: true);
+            for (var i = 0; i < count; i++) {
+              buffer.add(UuidValue.fromByteList(Uint8List.view(raw.buffer, i * 16, 16)));
+            }
+            return buffer;
```

Comment:
I'm such a big noob with Dart (or with every language? lol) but anyway, you guys @fzyzcjy @Desdaemon experts at Dart, I have a feeling that this is a code smell ... any recommendation on how to make it faster / more idiomatic Dart ?

Reply:
especially the part where I declare `List<UuidValue>.empty` usually that means that the list is not pre-allocated.

Reply:
apparently they mention about `List.generate`, I'm gonna try that.

Reply:
done in a9ada6d536ea96a37a0595fea41a2c83b556a06d